examples/platform/silabs/MatterConfig.cpp

```diff
@@ -376,6 +380,10 @@ CHIP_ERROR SilabsMatterConfig::InitMatter(const char * appName)
     Tracing::Register(backend);
 #endif // MATTER_TRACING_ENABLED
 
+#if defined(SILABS_TRACING_ENERGY_TRACES) && SILABS_TRACING_ENERGY_TRACES == 1
```

Comment:
Shouldn't this be inside the MATTER_TRACING_ENABLED ifdef? Could the power tracing work without the MATTER_TRACING_ENABLED?